news/4.34/jdt.html

```diff
@@ -134,6 +134,22 @@ <h2>Java Views and Dialogs</h2>
 			<img src="images/move-filtersbutton-to-top-other-way.png" alt="old menu">
 		  </p>
       </td>
+	  <tr id="experimental-preference">
+	  	  <td class="title"><strong>Experimental!</strong> Preference added to enable update on runtime behavior</a></td>
+		  <td class="content">
+              <p>
+				This update introduces a new preference setting that allows HiDPI support, enabling monitor-specific UI scaling for improved display clarity. Enabling this preference will also set the <strong>Edge</strong> browser as the default.
+				This check is disabled by default to streamline standard functionality. However, it can be activated if testing of this specific functionality is required. To enable, please configure the appropriate setting as described in the documentation.
+				<strong>Note: This preference check is only available in Windows</strong>
+  			</p>
+  			<p>
+  				To enable this check, navigate to Windows → Preferences → General → Appearance → HiDPI Settings, as shown in the image below.
```

Comment:
```suggestion
  				To enable this check, navigate to Window → Preferences → General → Appearance → HiDPI settings, as shown in the image below.
```

Maybe you can leave out the reference to "HiDPI settings" as one may search for a subpage of "Appearance" but it's only a group inside that page.